app/views/public_reviews/new_reviewer.html.haml

```diff
@@ -4,7 +4,7 @@
       %li= review.reviewer
 
 = simple_form_for @person, url: create_reviewer_public_reviews_path do |f|
-  = select_tag(:revision_id, options_from_collection_for_select(ArticleRevision.order('received desc').limit(20), :id, :title))
+  = select_tag "Artykuł (wersja)", :revision_id, options_from_collection_for_select(ArticleRevision.order('received desc').limit(20), :id, :title)
```

Comment:
To nie jest właściwe rozwiązanie - żeby to zadziałało trzeba dodać atrybut `label`, który będzie zawierał opis tego pola. Tę linijkę proszę przywrócić bo to nie zadziała.
http://apidock.com/rails/ActionView/Helpers/FormTagHelper/label_tag